exampleCourse/questions/fibonacciEditor/tests/initial_code.py

```diff
@@ -0,0 +1,2 @@
+def fib(n):
```

Comment:
Nit: the `tests` directory is really reserved for external grading support - could you put this file in the question root instead?

Reply:
should I change the example back to original format, where the code is inserted inside the element, and not from a file? In that case, I will just delete the file `initial-code.py`. Or do you think it is better to keep the source file as the example?

Reply:
I would just put two elements in that question, one sourced from a file, one sourced from the children of the element. The question doesn't necessarily have to make sense - I view them mostly as demos of how to use the code.

Reply:
ok!

Reply:
If I add two elements, would't that mess up the autograder?

Reply:
Ah, sorry, forgot which PR I'm in :) I would say to prefer the `source-file-name` attribute in this example.